pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -115,10 +115,14 @@ pub mod pallet {
 
 		/// The asset id of the collateral currency.
 		type CollateralAssetId: Get<CollateralAssetIdOf<Self>>;
-		/// Who can create new bonded currency pools.
+		/// The origin for most permissionless and priviledged operations.
+		type DefaultOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = Self::AccountId>;
```

Comment:
Where would this origin be used? For what extrinsics?

Reply:
For most of them, like mint_into, burn_into, start_destroy ... everything that's not creation or a force origin

Reply:
that's what I took from our conversations, that you wanted a configurable origin for all of these things?